graph/src/blockchain/mod.rs

```diff
@@ -247,6 +250,37 @@ impl From<web3::Error> for IngestorError {
     }
 }
 
+#[derive(Debug)]
+pub struct TriggerFilterWrapper<C: Blockchain> {
```

Comment:
Can you please add some comments on what this is, what it does and how it's supposed to be used?